src/docfx/build/Build.cs

```diff
@@ -148,7 +154,8 @@ void ValidateBookmarks()
                 switch (file.ContentType)
                 {
                     case ContentType.Resource:
-                        model = BuildResource(file);
+                        model = BuildResource.Build(file, monikersMap, referencingFiles);
```

Comment:
>Build [](start = 46, length = 5)

how about adding `monikers` to the return as `BuildPage`, then we can avoid casting `model`. Same for `BuildRedirection`